### ISSUE TITLE
Feature Request - Experiment Benchmark Outcome

### ISSUE THREAD

**Issue by David-Taub** [allegroai/clearml#745]
I would love it if we could log conclusive metric \ benchmark values, that indicate the performance of the experiment, e.g. benchmark outcome of the trained model on the test dataset. Currently, AFAIK, this is done by logging the values as a scalar, but this seems awkward since this is not a tracked value over epochs \ steps, but a singular value per experiment that unrelated to the batches \ iteration of the training. 

As I see it, this is the main way to trained models should be compared, along with some extremum point values that are tracked over the training such as min validation loss.

**Comment by ainoam** [comment]
Hi @David-Taub.

Isn't this the same as https://github.com/allegroai/clearml/issues/400?
This has been addressed by [Logger.report_single_value](https://github.com/allegroai/clearml/blob/fdefa9784cc4e0ffb9f0e01e0c3014cfdf489c5f/clearml/logger.py#L175) in [v1.5.0](https://github.com/allegroai/clearml/releases/tag/v1.5.0)

**Comment by ainoam** [comment]
@David-Taub Can we consider this issue closed then?

**Comment by David-Taub** [comment]
Yes, that does the trick. Thanks!
